packages/components/src/components/checkbox-wrapper/checkbox-wrapper-styles.ts

```diff
@@ -3,24 +3,36 @@ import { getCheckboxRadioJssStyle } from '../../styles/checkbox-radio-styles';
 import type { FormState } from '../../utils/form/form-state';
 import { getCss, mergeDeep } from '../../utils';
 import { getInlineSVGBackgroundImage } from '../../utils/svg/getInlineSVGBackgroundImage';
-import { addImportantToEachRule, getThemedColors } from '../../styles';
-import { borderRadiusMedium, borderRadiusSmall } from '@porsche-design-system/utilities-v2';
+import { addImportantToEachRule, getThemedColors, getTransition } from '../../styles';
+import { borderRadiusMedium, borderRadiusSmall, fontFamily, fontLineHeight } from '@porsche-design-system/utilities-v2';
 
 export const getComponentCss = (
   hideLabel: BreakpointCustomizable<boolean>,
   state: FormState,
   isDisabled: boolean,
+  isLoading: boolean,
+  isChecked: boolean,
```

Comment:
Perhaps we could make use of the `:checked` pseudo selector. https://developer.mozilla.org/en-US/docs/Web/CSS/:checked?retiredLocale=de

Reply:
@denyo in this case not, because we need to style spinner depending on the state of input

Reply:
@denyo thanks for the hint, I found a way to do it and changed the implementation